Description: use usr/share/ipython/notebook by default for the static data
 provided by ipython-notebook-common
 during build the folder does not exist so use the local tree path for tests
Author: Julian Taylor <jtaylor.debian@googlemail.com>
Forwarded: not-needed

--- a/IPython/html/__init__.py
+++ b/IPython/html/__init__.py
@@ -2,7 +2,10 @@
 
 import os
 # Packagers: modify this line if you store the notebook static files elsewhere
-DEFAULT_STATIC_FILES_PATH = os.path.join(os.path.dirname(__file__), "static")
+if os.path.exists("/usr/share/ipython/notebook/static"):
+    DEFAULT_STATIC_FILES_PATH = "/usr/share/ipython/notebook/static"
+else:
+    DEFAULT_STATIC_FILES_PATH = os.path.join(os.path.dirname(__file__), "static")
 
 # Packagers: modify the next line if you store the notebook template files
 # elsewhere
--- a/IPython/nbconvert/filters/marked.js
+++ b/IPython/nbconvert/filters/marked.js
@@ -3,7 +3,7 @@
 // that we use in the live notebook.
 
 // IPython static_path dir relative to here:
-var static_path = __dirname + "/../../html/static/";
+var static_path = "/usr/share/ipython/notebook/static/";
 
 // Excerpt from the example in require.js docs
 // http://requirejs.org/docs/node.html
